mmcv/transforms/formatting.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import warnings
+from typing import Optional, Sequence, Union
+
+import numpy as np
+import torch
+
+import mmcv
+from .base import BaseTransform
+from .builder import TRANSFORMS
+
+
+def to_tensor(
+    data: Union[torch.Tensor, np.ndarray, Sequence, int,
+                float]) -> torch.Tensor:
+    """Convert objects of various python types to :obj:`torch.Tensor`.
+
+    Supported types are: :class:`numpy.ndarray`, :class:`torch.Tensor`,
+    :class:`Sequence`, :class:`int` and :class:`float`.
+    Args:
+        data (torch.Tensor | numpy.ndarray | Sequence | int | float): Data to
+            be converted.
+
+    Returns:
+        torch.Tensor: the converted data.
+    """
+
+    if isinstance(data, torch.Tensor):
+        return data
+    elif isinstance(data, np.ndarray):
+        return torch.from_numpy(data)
+    elif isinstance(data, Sequence) and not mmcv.is_str(data):
+        return torch.tensor(data)
+    elif isinstance(data, int):
+        return torch.LongTensor([data])
+    elif isinstance(data, float):
+        return torch.FloatTensor([data])
+    else:
+        raise TypeError(f'type {type(data)} cannot be converted to tensor.')
+
+
+@TRANSFORMS.register_module()
+class ToTensor(BaseTransform):
+    """Convert some results to :obj:`torch.Tensor` by given keys.
+
+    Args:
```

Comment:
Indicate which key will be updated. We can say the keys in the argument ``keys``

Reply:
Give some example about supporting which kinds of key

Reply:
what' happened if  `results= {"instance": {"bbox": [[]], "label": []}}` and the parameter `keys = ["instance"]`

Reply:
It is expected that they can be converted to Tensor in a recursive manner.